lib/Service/OpenProjectAPIService.php

```diff
@@ -534,7 +547,8 @@ public static function validateIntegrationSetupInformation(array $values, bool $
 			'openproject_client_secret',
 			'default_enable_navigation',
 			'default_enable_unified_search',
-			'setup_group_folder'
+			'setup_group_folder',
```

Comment:
Maybe this key as well?
```suggestion
			'setup_project_folder',
```